pp/sidebarsPp.js

```diff
@@ -787,6 +787,10 @@ module.exports = {
 		  type: 'doc',
 		  id: 'integrations/plugin-packs/procedures/cloud-azure-storage-storagesync'
 		},
+        {
+          type: 'doc',
+          id: 'integrations/plugin-packs/procedures/cloud-azure-compute-vmscalesets'
+        },  
```

Comment:
Warning, indentation is probably misconfigured, make sur you have a tab = 4 spaces